docs/configure-vars-env.md

```diff
@@ -0,0 +1,24 @@
+# Configure my environment
+
+## Copy environment variables
+
+A `.env` file should be created in the repository root, to keep configuration default on the dev setup:
+
+```
+cp example.env .env
+```
+
+Replace `REPO_PATH_SERVER` with the path from above.
+
+## Setting the PHP version to be used
+
+The Nextcloud instance is setup to run with PHP 7.2 by default.
```

Comment:
I sure hope this info is obsolete.
Please update with actually used PHP version, and add 8.1 to the list of available versions below. (unless it is not yet supported but I think I tried it at some point)

Reply:
Yes, I changed that in this commit : https://github.com/juliushaertl/nextcloud-docker-dev/pull/86

I am rewriting this :+1:

Reply:
I fixed it in this commit : https://github.com/juliushaertl/nextcloud-docker-dev/commit/b501b22a3b9d9b23baf20c0d523d460bac789d44